src/System.Text.Json/tests/JsonTestHelper.cs

```diff
@@ -727,5 +727,12 @@ public static string GetCompactString(string jsonString)
 
         public static string StripWhitespace(this string value)
             => s_stripWhitespace.Replace(value, string.Empty);
+
+        public static string NormalizeLineEndings(this string value)
+        {
+            value = value.Replace("\r\n", "\n");
+            value = value.Replace('\r', '\n');
```

Comment:
Why is this second line needed?

Reply:
In case you checked out on OS which uses CR as line ending

Reply:
What OSes that developers actively use to run corefx tests use "\r" as a line ending?

Reply:
actually reading now, I'm not sure if there is such OS nowadays

Reply:
seems some of the tests already do normalization slightly differently so I have generalized the logic